### ISSUE TITLE
Highlights Cropper - Fifa, Cricket any sporting event

### ISSUE THREAD

**Issue by hiteshdua1** [open-source-ideas/open-source-ideas#95]
## Project description

This scripts chunks out videos of 4s, 6s and wickets of a live cricket match , so that I don't have to watch the entire match. These chunks can be further clubbed together to create specific highlights of a player or special moments of the matches.

Reduces a lot of your valuable time in watching the live match and dull moments 

## Relevant Technology

I created a POC using python and ffmpeg reference :  https://github.com/hiteshdua1/Cric-Cropper which can be further enhanced to be developed as a full project

## Who is this for

[_Describe who this project is suited for. Experienced developers, novice, or beginners?_]

## Complexity and required time

_Please only tick off one box in each category by changing `[ ]` to `[x]`. The labels on the project will then be updated by the maintainers as soon as possible._

### Complexity

- `[ ]` Beginner - This project requires no or little prior knowledge of the technolog(y|ies) specified to contribute to the project
- `[x]` Intermediate - The user should have some prior knowledge of the technolog(y|ies) to the point where they know how to use it, but not necessarily all the nooks and crannies of the technology
- `[ ]` Advanced - The project requires the user to have a good understanding of all components of the project to contribute

### Required time (ETA)

- `[ ]` Little work - A couple of days
- `[x]` Medium work - A week or two
- `[ ]` Much work - The project will take more than a couple of weeks and serious planning is required

---

Share on Twitter/other social media if you will. Something like

```
New project on Open Source Ideas Initiative: [fill in short description] – [fill in link]
```

**Comment by franckstifler** [comment]
I have an idea like this one, but what is the issues with copyrighting? Are you authorized to extract a video let us say from YouTube and paste on your website? What will then be the source of the video in that case, if is YouTube I have seen a website that can do it: https://hesetube.com/

**Comment by tvvignesh** [comment]
I saw something similar demonstrated in FB Hackathon here (See from 38:40): https://www.facebook.com/hackathon/videos/10153916963680988/

**Comment by hiteshdua1** [comment]
@franckstifler  You are right about the copyright issue, but what I am suggesting is to use the free live streaming services of some websites which have a delay of 5 minutes , so that i dont have to wait for the complete match to get over to get the highlight of that specific innings

As @tvvignesh  pointed out, I now see multiple use cases for this, and this can be factorized into multiple things ( live videos) and summarizing them. Although that is not what the original idea was.
